### PR TITLE
samples: net: Add documentation to prj_qemu_cortex_m3.conf file. 

### PR DESCRIPTION
**Pull request by m-kru**
For some qemu targets peer IP address and device address were swapt.

Signed-off-by: Michał Kruszewski <michal.kruszewski@nordicsemi.no>

### CODE REVIEW

**Comment by pfalcon** [comment]
Did you test all working modes after this change, e.g. https://www.zephyrproject.org/doc/subsystems/networking/qemu_setup.html?highlight=qemu#network-connection-between-two-qemu-vms ?

**Comment by m-kru** [comment]
@pfalcon No, I did not. I have just tested basic scenario with echo-server app.

File: samples/net/echo_client/prj_qemu_x86.conf [hunk 1]
```diff
@@ -33,5 +33,5 @@ CONFIG_NET_APP_SETTINGS=y
 # addresses (so that peer address is ending to 2).
 CONFIG_NET_APP_MY_IPV6_ADDR="2001:db8::2"
 CONFIG_NET_APP_PEER_IPV6_ADDR="2001:db8::1"
-CONFIG_NET_APP_MY_IPV4_ADDR="192.0.2.2"
-CONFIG_NET_APP_PEER_IPV4_ADDR="192.0.2.1"
+CONFIG_NET_APP_MY_IPV4_ADDR="192.0.2.1"
```
Comment (jukkar) [inline samples/net/echo_client/prj_qemu_x86.conf]: There was a reason for this swap for qemu_x86 and qemu_cortex_m3 (although this is not documented in arm conf file). See the comment few lines above, we have the addresses swapped so that we can simply test the qemu<->qemu communication.

File: samples/net/echo_client/src/tcp.c [hunk 2]
```diff
@@ -348,7 +348,7 @@ int start_tcp(void)
 				  K_THREAD_STACK_SIZEOF(
 					  net_app_tls_stack_ipv4));
 		if (ret < 0) {
-			NET_ERR("Cannot init IPv6 TCP client (%d)", ret);
+			NET_ERR("Cannot init IPv4 TCP client (%d)", ret);
```
Comment (jukkar) [inline samples/net/echo_client/src/tcp.c]: This is ok fix but should be in separate patch.

**Review by jukkar — CHANGES_REQUESTED** [review]
The address swap fix need to be declined at this point (unless we decide to change the use case for this file). If you wish, you could send a patch that adds the same comment to arm conf file as what is found in x86 conf file.

The error print fix in echo-client should be send separately and not combined to this patch.

File: samples/net/echo_client/prj_qemu_x86.conf (hunk 1 above)
Comment (m-kru) [reply samples/net/echo_client/prj_qemu_x86.conf]: @jukkar Ohh, sorry. I was using cortex_m3.conf I thought it was wrong then I just greped to see where is the similar situation without reading the rest of .conf file. I will change this patch to just improve documentation of prj_qemu_cortex_m3.conf

**Review by jukkar — APPROVED** [review]
